marketpulse/main/templates/fxosprice_all.html

```diff
@@ -74,6 +74,32 @@ <h3 class="title">Contributions</h3>
         {% endfor %}
       </tbody>
     </table>
+
+    <div class="row">
+      <div class="col-md-12 text-center">
+        {% if contributions.has_previous() %}
+          <a class="prev" title="Previous page" data-tooltip
+             href="{{ '#'|urlparams(page=contributions.previous_page_number()) }}">
+            &laquo;
+          </a>
+        {% endif %}
+        <label for="page-select">Page: </label>
+        <select id="page-select" autocomplete="off">
+          {% for i in range(1, contributions.paginator.num_pages + 1) %}
+            <option {% if i == contributions.number %} selected {% endif %}
+                    value="{{ '#'|urlparams(page=i) }}">
+              {{ i }}
+            </option>
+          {% endfor %}
+        </select>
```

Comment:
We could use [Bootstrap's pagination](http://getbootstrap.com/components/#pagination) here to look nicer. But I guess we can merge this and fix this on a second PR.